trajopt_ifopt/src/discrete_collision_evaluators.cpp

```diff
@@ -0,0 +1,149 @@
+/**
+ * @file discrete_collision_evaluators.h
```

Comment:
Not reviewing this since I assume it was just copied. Let me know if that isn't true

Reply:
It was, the only thing I changed is the collision buffer was not being applied to the contact manager and I have added a note asking if we should also add this to the link to link margin (I believe we should but wanted a second opinion).